docs/conf.py

```diff
@@ -103,10 +104,6 @@ def _get_project_meta() -> dict[str, str]:
 
 add_module_names = False
 
-autodoc_default_options = {
-    "show-inheritance": True,
-}
-
```

Comment:
Why was this removed? Do we not want to show inheritance anymore?

Reply:
I do not like to see `Bases: ...` on response classes, as this gives no info for the user (base classes are not a part of Public API).

Also, there is `:show-inheritance:` almost on every class.